components/prize_pool/commons/prize_pool_base.lua

```diff
@@ -82,13 +82,19 @@ BasePrizePool.config = {
 	storeSmw = {
 		default = true,
 		read = function(args)
-			return Logic.readBoolOrNil(args.storesmw)
+			return Logic.nilOr(
+				Logic.readBoolOrNil(args.storesmw),
+				not Logic.readBool(Variables.varDefault('disable_LPDB_storage'))
+			)
 		end
 	},
 	storeLpdb = {
 		default = true,
 		read = function(args)
-			return Logic.readBoolOrNil(args.storelpdb)
+			return Logic.nilOr(
+				Logic.readBoolOrNil(args.storelpdb),
+				not Logic.readBool(Variables.varDefault('disable_LPDB_storage'))
```

Comment:
This will ignore the `default` value, as the 2nd param will always return true/false

Reply:
Right, so it would have to be something like
```suggestion
				Logic.readBoolOrNil(Variables.varDefault('disable_LPDB_storage')) ~= nil and (not Logic.readBool(Variables.varDefault('disable_LPDB_storage'))) or nil
```